### ISSUE TITLE
Infiltration: The AI hijack module does not warn the AI of being converted.

### ISSUE THREAD

**Issue by TheTrigg** [HippieStation/HippieStationdeprecated2020#12082]
At start, I didn't know what the hijack module did. All I knew was that I needed to put it in the AI core for the objective to count.
Problem was, it started screaming on the radio, warning the crew of our presence. We had to card it so it wouldn't, then put it back into a built and hidden AI core right before roundend.

But then.. After roundend.. These were its laws.
![image](https://user-images.githubusercontent.com/36010999/65364786-494b1b80-dbea-11e9-85ba-cf7edee5f368.png)

So it was supposed to be on our side. I ahelped and reported it to an admin, the admin talked to the player and came back to me saying that the player didn't recieve any indication of being converted to the syndie side. I guess he could have been lying.

So, I am not actually certain that this is a bug at all. But, I could test it to make sure. But for that...
I need someone else. And I'm a lonely man.

[Reproduction]: Have an AI, have an infiltrator (so that makes 2 players), use the hijack module on the AI and see if anything pops up for them.

**TL;DR:** It may or may not actually happen. I cannot test it for myself because I need friends.

**Comment by ghost** [comment]
```
/datum/antagonist/hijacked_ai/greet()
	to_chat(owner, "<span class='userdanger'>You have been hijacked!</span>")
	to_chat(owner, "<span class='danger bold'>A Syndicate agent has successfully deployed a SEU attack on you. <I>You are now utterly loyal to the cause of the syndicate.</I></span>")
	to_chat(owner, "<span class='danger bold'>You feel your power expand as the exploitation unit gives you a new interface.</span>")
```

it literally should display "You have been hijacked!" in BIG RED LETTERS.

sorry bud, can't fix players not looking at chat

**Comment by TheTrigg** [comment]
The man was lying, then. Darn. I'll close this.
